kanidm_unix_int/pam_kanidm/build.rs

```diff
@@ -0,0 +1,6 @@
+fn main() {
+    pkg_config::Config::new()
+        .atleast_version("1.3.0")
```

Comment:
The minimum pam version was chosen somewhat arbitrarily.

I checked packaged versions from https://repology.org/project/pam/versions and just picked the number openSUSE has under the assumption that's the version that the code's been tested with.

Reply:
Checks out from deb-ish land:

- Debian 10: 1.3.1
- Ubuntu 21.10: 1.3.1